src/main/resources/Message_fr.properties

```diff
@@ -4,7 +4,10 @@ ACCEPT_DEPENDENT_EMAIL_LINK_TEXT = Accepter la demande
 ACCEPT_DEPENDENT_EMAIL_SUBJECT = Demande d'accompagnateur
 ACCEPT_DEPENDENT_EMAIL_MANAGE = G�rez vos pr�f�rences
 ACCEPT_DEPENDENT_ERROR = La demande d'accompagnateur n'a pas �t� re�ue.
+ARRIVED_NOTIFICATION = TODO %s.
```

Comment:
@binh-dam-ibigroup can you provide the french messages please!

Reply:
Oui ! https://github.com/ibi-group/otp-middleware/pull/275/commits/36b26afa7d532112d72346b040408576db40a705